Resources/Textures/Shaders/brightnessfilter.swsl

```diff
@@ -0,0 +1,9 @@
+uniform highp float THRESHHOLD;
+uniform sampler2D SCREEN_TEXTURE;
+
+void fragment() {
+    highp vec4 THISCOLOR = texture(SCREEN_TEXTURE, UV);
+    COLOR = vec4(1, 1, 1, 1);
```

Comment:
This isn't going to compile on drivers that strictly comply with the GLES spec (such as the engine's built-in compatibility mode, which uses ANGLE, a compatibility layer that has very strict compliance with the GLES spec). These need to be made explicit floats, as implicit conversions from integer to float simply do not exist in the GLES spec.

```suggestion
    COLOR = vec4(1.0, 1.0, 1.0, 1.0);
```